lib/python/qmk/cli/new/keyboard.py

```diff
@@ -0,0 +1,98 @@
+"""This script automates the generation of a new keyboard directory.
+"""
+import os
+import shutil
+
+from distutils.dir_util import copy_tree
+from datetime import datetime
+from milc import cli
+
+
+def replace_files(filenames, search, replace):
+    """Replaces keywords across multiple files
+    """
+    for filename in filenames:
+        with open(filename, 'r') as file:
+            contents = file.read()
+
+        contents = contents.replace(search, replace)
+
+        with open(filename, 'w') as file:
+            file.write(contents)
```

Comment:
This could be done with only 1 file opening:

```
with open(filename, 'r+') as file:
    contents = file.read()
    file.seek(0)
    file.write(contents.replace(search, replace))
    file.truncate()
```
But the performance difference is non-existent in this use-case, so this is more like an idea than anything else.
Let's see how @skullydazed feels about it.

Reply:
I like that, it's cleaner.